src/Util/FilterUtil.ts

```diff
@@ -2,9 +2,18 @@ import {
   Filter,
   Operator,
   CombinationOperator,
-  NegationOpertaor
+  NegationOpertaor,
```

Comment:
Not really your change, but this is definitely a typo

Reply:
True, this needs to be fixed at first in geostyler-style and then on all other packages using this type

Reply:
AFAICT I fixed this in a PR...

Reply:
https://github.com/terrestris/geostyler-style/pull/80